TeamMemberApi/members/views.py

```diff
@@ -0,0 +1,83 @@
+from django.http import HttpResponse, JsonResponse
+from django.template import loader
+from rest_framework.renderers import JSONRenderer
+from rest_framework.parsers import JSONParser
+from members.models import Member
+from members.serializers import MemberSerializer
+from django.views.decorators.csrf import csrf_exempt, csrf_protect
+from django.views import generic
+from django.views.decorators.csrf import ensure_csrf_cookie
+
+"""
+    WARNING: Cross site requests not allowed. You can
+    add @csrf_exempt decorator above functions below
+    for testing purposes
+"""
+
+@csrf_protect
+def member_list(request):
```

Comment:
Also, I believe DRF's views are automatically exempt from CSRF protection. Generally, API views are authenticated with a token in the header rather than a browser cookie, so CSRF protection is not required.